src/components/tiles/tile-navigator.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { CSSProperties, useMemo, useRef } from "react";
+import { observer } from "mobx-react-lite";
+import useResizeObserver from "use-resize-observer";
+
+import { ITileProps } from "./tile-component";
+
+import NavigatorMoveIcon from "../../assets/icons/navigator-move-icon.svg";
+
+import "./tile-navigator.scss";
+
+interface INavigatorProps {
+  children: React.ReactElement<ITileProps>;
+}
+
+const navigatorSize = { width: 90, height: 62 };
+
+export const TileNavigator = observer(function TileNavigator({children}: INavigatorProps) {
```

Comment:
Since this is a shared component, a comment explaining what it is would be helpful.